feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/item/AboutIconItem.kt

```diff
@@ -0,0 +1,29 @@
+package io.github.droidkaigi.confsched2020.about.ui.item
+
+import com.squareup.inject.assisted.Assisted
+import com.squareup.inject.assisted.AssistedInject
+import com.xwray.groupie.databinding.BindableItem
+import io.github.droidkaigi.confsched2020.about.R
+import io.github.droidkaigi.confsched2020.about.databinding.ItemAboutIconBinding
+
+class AboutIconItem @AssistedInject constructor(
```

Comment:
I am struggling with naming "AboutIconItem" or "AboutLaunchItem".
What do you think about it?

Reply:
@umechanhika Hi, It's good opinion! `AboutLaunchItem` would be nice. However this p-r works well 😄 I'm going to merge #702 . 

if you are still interested, Could you make a change p-r where applicable `AboutIconItem` to `AboutLaunchItem` ?